lighthouse-core/report/handlebar-helpers.js

```diff
@@ -63,10 +63,9 @@ const handlebarHelpers = {
     return calculateRating(totalScore);
   },
 
-  // Converts an aggregation's score to a rating used for styling.
+  // Converts an aggregation's score to a pass/fail rating.
   getAggregationScoreRating: score => {
-    const rating = calculateRating(score * 100);
-    return rating !== 'good' ? 'poor' : rating; // avg rating maps to failure for aggregation.
+    return score === 1 ? 'good' : 'poor';
```

Comment:
yes. :)  I have a feeling that `calculateRating` is returning ~0.9 in that case.. thus the question. :)